batch10/assignment2/syed-bilal.md

```diff
@@ -0,0 +1,4 @@
+2. VM has Hardware Level Virtualization where as Containers have OS Level Virtualization
+3. Docker uses a client-server architecture. The Docker client talks to the Docker daemon, which does the heavy lifting of building, running, and distributing your Docker containers.
+4. docker container run -d --publish 9090:80 nginx --name assignment-2
```

Comment:
The image name should always be at the end.... otherwise it will be considered as CMD
docker container run -d --publish 9090:80 --name assignment-2 nginx